hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFuture.java

```diff
@@ -144,8 +144,9 @@ public void run() {
      * 'set' calls are ignored.
      *
      * @param offeredResponse The type of response to offer.
+     * @return <tt>true</tt> is offered response is set, <tt>false</tt> otherwise.
```

Comment:
Can you add a bit more explanation of what 'set' means. The issue here is the 'WAIT_RESPONSE'; it is set.. but it isn't the final response for the invocation.